titanic/README.md

```diff
@@ -0,0 +1,163 @@
+# TITANIC CLASSIFICATION EXAMPLE
```

Comment:
The main README.md should have a link to titanic https://github.com/pallavi-pannu-oc/dkube-examples/tree/7bdc670ebfc7f129baa87824ff709f49010b05c4